README.md

```diff
@@ -107,6 +107,8 @@ The `data` parameter in `callback` is defined as follow:
   either fully consume or destroy the body unless there is an error, or no further requests
   will be processed.
 
+`headers` is an object where all keys has been lowercased.
```

Comment:
Nit :)

```suggestion
`headers` is an object where all keys have been lowercased.
```

Reply:
You did not see this. It never happened. :smile: